spec/integ/crypto/crypto.spec.ts

```diff
@@ -2385,6 +2450,96 @@ describe.each(Object.entries(CRYPTO_BACKENDS))("crypto (%s)", (backend: string,
             expect(userSigningKey[secretStorageKey]).toBeDefined();
             expect(selfSigningKey[secretStorageKey]).toBeDefined();
         });
+
+        oldBackendOnly("should create a new megolm backup", async () => {
+            const backupVersion = "abc";
+            await bootstrapSecurity(backupVersion);
+
+            // Expect a backup to be available and used
+            const activeBackup = await aliceClient.getCrypto()!.getActiveSessionBackupVersion();
+            expect(activeBackup).toStrictEqual(backupVersion);
+        });
+
+        oldBackendOnly("Reset key backup should create a new backup and update 4S", async () => {
```

Comment:
This isn't testing `bootstrapSecretStorage`, so please move it outside the `describe("bootstrapSecretStorage",...)` block.

Reply:
It tightly related, the secret storage should stays in sync when reseting the backup. It shares a lot of common setup code. Moving it outside would require to move also all setup function somewhere.

Reply:
> It tightly related, the secret storage should stay in sync when reseting the backup.

It might be related, but I don't accept that it's testing `bootstrapSecretStorage`. It assumes that `bootstrapSecretStorage` works as expected.

Alternatively expressed: we should make sure that we are clear which tests are for testing which method. We should avoid having big tests which test _lots of things_ because they make it hard to see what has broken when they fail.

> It shares a lot of common setup code. Moving it outside would require to move also all setup function somewhere.

We shouldn't be afraid to clean up code. Reluctance to do so in the past is one of the reasons this entire Element R project has taken 3 times as long as we expected: nobody understands how the js-sdk works any more.

Reply:
Created a new task to handle that after the 3 backup PRs are merged to simplify the refactoring